catalog-rest-service/src/main/resources/json/operations/workflows/ingestion.json

```diff
@@ -0,0 +1,171 @@
+  "title": "IngestionConfig",
+  "description": "Ingestion Config is used to setup a Airflow Ingestion pipeline.",
+  "type": "object",
+  "definitions": {
+    "ingestionType": {
+      "description": "Type of Ingestion - Bigquery, Redshift, Snowflake etc...",
+      "type": "string",
+      "enum": [
+        "bigquery",
+        "bigquery-usage",
+        "redshift",
+        "redshift-usage",
+        "snowflake",
+        "snowfalke-usage",
+        "hive",
+        "mssql",
+        "mysql",
+        "postgres",
+        "trino",
+        "vertica"
+      ],
+      "javaEnums": [
+        {
+          "name": "bigquery"
+        },
+        {
+          "name": "bigquery-usage"
+        },
+        {
+          "name": "redshift"
+        },
+        {
+          "name": "redshift-usage"
+        },
+        {
+          "name": "snowflake"
+        },
+        {
+          "name": "snowflake-usage"
+        },
+        {
+          "name": "hive"
+        },
+        {
+          "name": "mssql"
+        }, {
+          "name": "looker"
+        },
+        {
+          "name": "presto"
+        },
+        {
+          "name": "postgres"
+        },
+        {
+          "name": "redash"
+        },
+        {
+          "name": "salesforce"
+        },
+        {
+          "name": "superset"
+        },
+        {
+          "name": "trino"
+        },
+        {
+          "name": "vertica"
+        }
+      ]
+    }
+  },
+  "properties" : {
+    "name": {
+      "description": "Name that identifies this ingestion instance uniquely.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256
+    },
+    "description": {
+      "description": "Description of the workflow.",
+      "type": "string"
+    },
+    "forceDeploy": {
+      "description": "Deploy the workflow by overwriting existing workflow with the same name.",
+      "type": "boolean",
+      "default": "false"
+    },
+    "pauseWorkflow": {
+      "description": "pause the workflow from running once the deploy is finished successfully.",
+      "type": "boolean",
+      "default": "false"
+    },
+    "concurrency": {
+      "description": "Concurrency of the Pipeline.",
+      "type": "integer",
+      "default": 1
+    },
+    "startDate": {
+      "description": "Start date of the workflow.",
+      "type": "string"
+    },
+    "endDate": {
+      "description": "End Date of the workflow.",
+      "type": "string"
+    },
+    "workflowTimezone": {
+      "description": "Timezone in which workflow going to be scheduled.",
+      "type": "string",
+      "default": "UTC"
+    },
+    "retries": {
+      "description": "Retry workflow in case of failure",
+      "type": "integer",
+      "default": 1
+    },
+    "retryDelay": {
+      "description": "Delay between retries in seconds.",
+      "type": "integer",
+      "default": 300
+    },
+    "workflowCatchup": {
+      "description": "Workflow catchup for past executions.",
```

Comment:
What does "catchup" refer to for ingestion?

Reply:
catchup as in , run for past days if the start date is scheduled for few days back
this will be useful in usage related workflows where we need to analyze the queries.

Reply:
Thanks for describing this. 

I believe another common term for this is backfill.

May be a better description (since this goes into documentation):
"Run past executions if the start date is in the past" (describes the term catchup or backfill, without using the term themselves)